app/screens/settings/settings-screen.tsx

```diff
@@ -41,7 +41,7 @@ export function SettingsScreen({}: SettingsScreenProps) {
 
   return (
     <Screen style={ROOT} preset="scroll">
-      {tracksDirectoryInfo ? (
+      {tracksDirectoryInfo?.exists ? (
         <Text
           text={`Der Speicherbedarf aller Downloads beträgt ${Math.round(
             tracksDirectoryInfo.size / 1000000,
```

Comment:
`size` only exists if `exists === true` (`FileSystem.FileInfo` is a union type discriminated using the `exists` flag:

```ts
export declare type FileInfo = {
    exists: true;
    uri: string;
    size: number;
    isDirectory: boolean;
    modificationTime: number;
    md5?: string;
} | {
    exists: false;
    uri: string;
    size: undefined;
    isDirectory: false;
    modificationTime: undefined;
    md5: undefined;
};
```